<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1081</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Set logging</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="version">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>161</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Action</string>
    </property>
   </widget>
   <widget class="QListWidget" name="topicField">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>221</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="package_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>221</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Topics</string>
    </property>
   </widget>
   <widget class="QPushButton" name="refreshButton">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>10</y>
      <width>88</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
   <widget class="QListWidget" name="actionField">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>40</y>
      <width>161</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="accountButton">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>10</y>
      <width>88</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Add account</string>
    </property>
   </widget>
   <widget class="QPushButton" name="enableButton">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>50</y>
      <width>88</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Enable</string>
    </property>
   </widget>
   <widget class="QPushButton" name="disableButton">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>90</y>
      <width>88</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Disable</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeButton">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>130</y>
      <width>88</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
   <widget class="QPushButton" name="backupButton">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>50</y>
      <width>88</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Add backup</string>
    </property>
   </widget>
   <widget class="QPushButton" name="certButton">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>90</y>
      <width>88</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Add certificate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="debugButton">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>130</y>
      <width>88</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Add debug</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dnsButton">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>170</y>
      <width>88</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Add DNS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="errorButton">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>210</y>
      <width>88</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Add error</string>
    </property>
   </widget>
   <widget class="QPushButton" name="eventButton">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>250</y>
      <width>88</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Add event</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fwButton">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>290</y>
      <width>88</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Add firewall</string>
    </property>
   </widget>
   <widget class="QPushButton" name="infoButton">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>10</y>
      <width>88</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Add info</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ipsecButton">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>50</y>
      <width>88</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Add ipsec</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ospfButton">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>90</y>
      <width>88</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Add OSPF</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ovpnButton">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>130</y>
      <width>88</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Add OVPN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="packetButton">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>170</y>
      <width>88</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Add packet</string>
    </property>
   </widget>
   <widget class="QPushButton" name="routeButton">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>210</y>
      <width>88</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Add route</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sshButton">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>250</y>
      <width>88</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Add ssh</string>
    </property>
   </widget>
   <widget class="QPushButton" name="wirelessButton">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>290</y>
      <width>88</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Add wireless</string>
    </property>
   </widget>
   <widget class="QListWidget" name="disableField">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>40</y>
      <width>161</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="version_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>161</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Disabled</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="version_3">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>10</y>
      <width>161</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Default</string>
    </property>
   </widget>
   <widget class="QListWidget" name="defaultField">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>40</y>
      <width>161</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
